cypress/integration/metaSpec.js

```diff
@@ -8,15 +8,16 @@ describe('On a page with meta tag imgix paramaters', () => {
       cy.fixture('config.js').as('config');
     });
     it('All loaded and rendered', () => {
+      // expect the number of image elements on the page to equal fixture's
+      cy.get('img').should('have.length', 3);
       cy.get('img').each(($el, idx, $list) => {
-        // epect the number of image elements on the page to equal fixture's
-        idx === 0 && expect($list.length).to.equal(3);
         // "naturalWidth" and "naturalHeight" are set when the image loads
         expect($el[0].naturalWidth).to.be.greaterThan(0);
       });
     });
 
     it('Interacted with the imgix instance', () => {
+      cy.get('img').should('have.length', 3);
```

Comment:
You probably want to have a similar check above line 38 for that test 👍

Reply:
Yup, just going to move this check to the beforeEach hook so that it runs on every test 👌🏼